arrow/formatter.py

```diff
@@ -116,7 +123,10 @@ def _format_token(self, dt, token):
         if token in ["ZZ", "Z"]:
             separator = ":" if token == "ZZ" else ""
             tz = dateutil_tz.tzutc() if dt.tzinfo is None else dt.tzinfo
-            total_minutes = int(util.total_seconds(tz.utcoffset(dt)) / 60)
+            # FIXME: utcoffset() is nullable
+            total_minutes = int(
+                util.total_seconds(cast(timedelta, tz.utcoffset(dt))) / 60
+            )
```

Comment:
I think we should refactor this code.
`tz.utcoffset(dt)` is nullable

Reply:
Go for it! Have anything in mind?

Reply:
Please review refactored code!